packages/riipen-ui/src/components/Avatar.jsx

```diff
@@ -96,6 +98,13 @@ class Avatar extends React.Component {
           .circle {
             border-radius: 50%;
           }
+
+          .image {
+            display: block;
+            height: calc(${size} + ${this.getBorderSize()} * 2);
+            width: calc(${size} + ${this.getBorderSize()} * 2);
```

Comment:
Have to take border into account.

Reply:
It makes me slightly nervous that this is `display: block`. I think as a user of the UI kit this would be unexpected. I am not entirely sure what this PR is really solving so I don't have any alternative suggestions.

@ksmort maybe can you elaborate a bit more on this?

Reply:
@jordanell Updated PR description and updated css in Avatar component, hopefully it makes more sense!